Sources/Danger/BitBucketCloud.swift

```diff
@@ -2,227 +2,243 @@ import Foundation
+        public let description: String
+
+        /// The PR's destination
+        public let destination: MergeRef
+
+        /// PR's ID
+        public let id: Int // swiftlint:disable:this identifier_name
+
+        /// People who have participated in the PR
+        public let participants: [Participant]
+
+        /// People requested as reviewers
+        public let reviewers: [User]
+
+        /// The PR's source, The repo Danger is running on
+        public let source: MergeRef
+
+        /// The pull request's current status.
+        public let state: State
+
+        /// Title of the pull request
+        public let title: String
+
+        /// Date of last update
+        public let updatedOn: Date
     }
+}
 
-    private enum CodingKeys: String, CodingKey {
-        case author
-        case createdOn = "created_on"
-        case description
-        case destination
-        case id // swiftlint:disable:this identifier_name
-        case participants
-        case reviewers
-        case source
-        case state
-        case title
-        case updatedOn = "updated_on"
-    }
-
-    /// The creator of the PR
-    public let author: BitBucketCloudUser
-
-    /// Date when PR was created
-    public let createdOn: Date
-
-    /// The text describing the PR
-    public let description: String
-
-    /// The PR's destination
-    public let destination: BitBucketCloudMergeRef
-
-    /// PR's ID
-    public let id: Int // swiftlint:disable:this identifier_name
-
-    /// People who have participated in the PR
-    public let participants: [BitBucketCloudPRParticipant]
-
-    /// People requested as reviewers
-    public let reviewers: [BitBucketCloudUser]
+extension BitBucketCloud {
+    public struct MergeRef: Decodable {
+        private struct Branch: Decodable, Equatable {
+            let name: String
+        }
 
-    /// The PR's source, The repo Danger is running on
-    public let source: BitBucketCloudMergeRef
+        private struct Commit: Decodable, Equatable {
+            let hash: String
+        }
 
-    /// The pull request's current status.
-    public let state: State
+        private let branch: Branch
 
-    /// Title of the pull request
-    public let title: String
+        public var branchName: String {
+            return branch.name
+        }
 
-    /// Date of last update
-    public let updatedOn: Date
-}
+        private let commit: Commit
 
-public struct BitBucketCloudMergeRef: Decodable {
-    private struct Branch: Decodable, Equatable {
-        let name: String
-    }
+        /// Hash of the last commit
+        public var commitHash: String {
+            return commit.hash
+        }
 
-    private struct Commit: Decodable, Equatable {
-        let hash: String
+        public let repository: Repo
     }
+}
 
-    private let branch: Branch
+extension BitBucketCloud {
+    public struct Repo: Decodable, Equatable {
+        private enum CodingKeys: String, CodingKey {
```

Comment:
<!--
  0 failure: 
  1 warning:  Types should be n...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketCloud.swift;
  Line: 123;
-->

- :warning: Types should be nested at most 1 level deep (`nesting`)